tutorials/backend/hasura-v3-ts-connector/tutorial-site/content/twitter-share.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const TwitterShare = () => {
+    return(
+        <React.Fragment>
+            <a href="https://twitter.com/intent/tweet?&text=I just completed this Hasura backend course for frontend developers by @HasuraHQ. Check it out here - https://hasura.io/learn/graphql/hasura/introduction/" target="_blank" rel="noopener noreferrer">
```

Comment:
As with the `config.js`, let's update the information here.

Reply:
Done